src/main.cpp

```diff
@@ -6081,6 +6081,25 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
                   pfrom->cleanSubVer, pfrom->nVersion,
                   pfrom->nStartingHeight, addrMe.ToString(), pfrom->id,
                   remoteAddr);
+	    
+        std::vector<std::string> vBannedVersions = mapMultiArgs["-banversion"];
+        bool fBanned = false;
+
+        for (unsigned int i = 0; i <= vBannedVersions.size(); i++)
```

Comment:
I think we should ban like this instead:
`if nodeVer <= bannedVer`
Not sure how we're representing versions in the codebase 
If it's like this `04002001` it should be easy to make this change)